system/Database/MigrationRunner.php

```diff
@@ -138,10 +138,7 @@ public function __construct(MigrationsConfig $config, $db = null)
         // Default name space is the app namespace
         $this->namespace = APP_NAMESPACE;
 
-        // get default database group
-        $config      = config(Database::class);
-        $this->group = $config->defaultGroup;
-        unset($config);
+        $this->group = is_string($db) ? $db : config(Database::class)->defaultGroup;
```

Comment:
It seems to me that we should include a comment here.

```suggestion
        // Even if a DB connection object is passed, since it is a test,
        // it is assumed to be the default group name.
        $this->group = is_string($db) ? $db : config(Database::class)->defaultGroup;
```